MIGRATION.md

```diff
@@ -0,0 +1,9 @@
+# Breaking changes
```

Comment:
Please be explicit about the version that is introducing those breaking changes and from which version we should migrate.

Proposal to make it the v1 of the lib.